django_lesson/lesson1/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.7
+
+RUN pip install django
```

апка не запускається, оскільки не всі залежності встановлено

---

```suggestion
RUN pip install -r requirements.txt
```
але перед цим потрібно спочатку створити requirements.txt, а потім запхати в докер

---

Done